test/e2e/tests/confirmations/signatures/sign-typed-data-v3.spec.ts

```diff
@@ -102,14 +146,16 @@ async function assertInfoValues(driver: Driver) {
 
 async function assertVerifiedResults(driver: Driver, publicAddress: string) {
   await driver.waitUntilXWindowHandles(2);
-  await driver.switchToWindowWithTitle(WINDOW_TITLES.TestDApp);
+  const windowHandles = await driver.getAllWindowHandles();
+  await driver.switchToWindowWithTitle('E2E Test Dapp', windowHandles);
   await driver.clickElement('#signTypedDataV3Verify');
+  await driver.delay(500);
 
   const verifyResult = await driver.findElement('#signTypedDataV3Result');
-  await driver.waitForSelector({
-    css: '#signTypedDataV3VerifyResult',
-    text: publicAddress,
-  });
+  // await driver.waitForSelector({
```

Comment:
remove?

Reply:
ah I forgot to remove the commented code, will update, thanks

Reply:
done